cpp/include/cugraph/algorithms.hpp

```diff
@@ -335,9 +331,8 @@ void edge_betweenness_centrality(const raft::handle_t& handle,
  * @param graph_view Graph view object.
  * @param edge_weight_view Optional view object holding edge weights for @p graph_view. Currently,
  * edge_weight_view.has_value() should be false as we don't support weighted graphs, yet.
- * @param vertices Optional, if specified this provides either a vertex_t count of how many
- *         random seeds to select, or a device_span identifying a list of pre-selected vertices
- *         to use as seeds for the traversals for approximating betweenness.
+ * @param vertices Optional, if specified this provides a device_span identifying a list of
+ * pre-selected vertices to use as seeds for the traversals for approximating betweenness.
  * @param normalized         A flag indicating results should be normalized
```

Comment:
It looks like this parameter is not working. I ran the CAPI tests with `normalized=True` and it is still passing

Reply:
Added implementation of normalization in latest push